core/src/test/java/io/questdb/test/griffin/DistinctTimeSeriesTest.java

```diff
@@ -204,4 +204,256 @@ public void testTimestampDescOrder() throws Exception {
             );
         });
     }
+
+    @Test
+    public void testOrderByTimestampIsPushedDownForDistinctTimestamp() throws Exception {
+        assertMemoryLeak(() -> {
+            ddl("create table tab (x int, y long, ts timestamp) timestamp(ts)");
+
+            assertPlan(
```

Comment:
nit: would be great to assert result sets, not only execution plans in these tests.